### PR TITLE
Should EnumType accept internal values when using enum.Enum?

### PR DESCRIPTION
**Pull request by jose-humu**
Resolving following issue: https://github.com/mirumee/ariadne/issues/200

### CODE REVIEW

File: tests/test_enums.py [hunk 1]
```diff
@@ -185,3 +185,33 @@ def test_int_enum_arg_is_transformed_to_internal_value():
...
+    query.set_field("testEnum", lambda *_: PyStringEnum.NEWHOPE)
+
+    schema = make_executable_schema([enum_definition, enum_field], [query, py_string_enum])
+    result = graphql_sync(schema, "{ testEnum }")
+    assert result.data["testEnum"] == TEST_VALUE
+
+
+def test_enum_string_arg_is_transformed_to_internal_value():
+    query = QueryType()
+    query.set_field("testEnum", lambda *_, value: value == PyStringEnum.NEWHOPE)
```
Comment (patrys) [inline tests/test_enums.py]: You should be able to use identity here instead of equality, enums are singletons.

File: tests/test_enums.py [hunk 2]
```diff
@@ -185,3 +185,33 @@ def test_int_enum_arg_is_transformed_to_internal_value():
...
+py_string_enum = EnumType("Episode", PyStringEnum)
+
+
+def test_enum_string__is_resolved_from_internal_value():
+    query = QueryType()
+    query.set_field("testEnum", lambda *_: PyStringEnum.NEWHOPE)
+
+    schema = make_executable_schema([enum_definition, enum_field], [query, py_string_enum])
+    result = graphql_sync(schema, "{ testEnum }")
+    assert result.data["testEnum"] == TEST_VALUE
```
Comment (patrys) [inline tests/test_enums.py]: Could we compare this against `PyStringEnum.NEWHOPE`? I hope to kill `TEST_VALUE` one day as it makes tests unreadable.

File: tests/test_enums.py [hunk 3]
```diff
@@ -185,3 +185,33 @@ def test_int_enum_arg_is_transformed_to_internal_value():
...
+class PyStringEnum(Enum):
+    NEWHOPE = "1977"
+    EMPIRE = "1980"
+    JEDI = "1983"
+
+
+py_string_enum = EnumType("Episode", PyStringEnum)
+
+
+def test_enum_string__is_resolved_from_internal_value():
```
Comment (patrys) [inline tests/test_enums.py]: Typo: double underscore

**Comment by rafalp** [comment]
Thank you for your PR!  We've investigated this and found a solution that doesn't require us for modifying our `EnumType` implementation. <3

**Pull request closed without merging**